### PR TITLE
Added 2 More models in addition to simple CNN

### PR DESCRIPTION
**Pull request by SourinRakshit-ai**
I have added a slightly better CNN model and a model using EfficientNetV2_S. 
The EfficientNet model is giving accuracy of around 97.08%

Screenshot 2025-05-06 at 11 40 39 PM

### CODE REVIEW

**Comment by david-boules** [comment]
Hey Sourin. My apologies for the late reply, I've just seen this now.
Nice work experimenting with both a model from scratch and some transfer learning approach, performance looks pretty solid overall. I would just recommend you also track some of the validation metrics (so validation loss, accuracy, etc.), so you can compare them with the training metrics and make sure the model is not overfitting. If validation loss or other metrics are very unstable, then some model tuning may be required. But great work overall, would love for you to share your progress with us during our next meeting as your input would be highly valuable to the team.
